### PR TITLE
adds 1st order everest's pld

### PR DESCRIPTION
**Pull request by mirca**
cc @barentsen

### CODE REVIEW

File: pyke/lightcurve.py [hunk 1]
```diff
@@ -140,3 +140,28 @@ def detrend(self):
 class ArcLengthDetrender(Detrender):
     def detrend(time, flux):
         pass
+
+
+class EverestDetrender(Detrender):
```
Comment (barentsen) [inline pyke/lightcurve.py]: We're not actually implementing Everest, so I would call this e.g. `SimplePixelLevelDetrender`.  The docstring should of course give credit to the Wang and Luger papers that explain the idea, and mention everest!

**Comment by barentsen** [comment]
:+1:  Does it work?  =)

**Comment by mirca** [comment]
screen shot 2017-10-12 at 10 45 57 am

**Comment by barentsen** [comment]
That's an improvement!!

**Comment by mirca** [comment]
cc @rodluger

**Comment by mirca** [comment]
cc @gully

**Comment by mirca** [comment]
I'm merging this since @christinahedges is working on something related.

**Comment by mirca** [comment]
cc @gully

**Pull request merged**